api/models.py

```diff
@@ -138,14 +138,24 @@ def compute_scores(self):
 
         logger.debug("Articles found %s", related_articles)
 
-        self._compute_content_score(counter_nouns_article, related_articles)
+        counter_article = 0
+        for word in counter_nouns_article:
+            if counter_nouns_article[word] > 1:
+                counter_article += 1
+        logger.debug("Number of interesting nouns : %s", counter_article)
+
+        if counter_article > 0:
+            self._compute_content_score(counter_nouns_article, related_articles, counter_article)
+        else:
+            self.delete()
+            return "Notre méthode de calcul n'a pas fournir de résultat sur cet article."
```

Comment:
Il manque un petit mot ;)
```suggestion
            return "Notre méthode de calcul n'a pas pu fournir de résultat sur cet article."
```